test/e2e/util.go

```diff
@@ -1377,7 +1378,6 @@ func (b kubectlBuilder) exec() (string, error) {
 		b.cmd.Process.Kill()
 		return "", fmt.Errorf("Timed out waiting for command %v:\nCommand stdout:\n%v\nstderr:\n%v\n", cmd, cmd.Stdout, cmd.Stderr)
 	}
-	Logf("stdout: %q", stdout.String())
```

Comment:
It's major spam because it logs 150 endpoints. I'm logging it in the "die" equivalent of this method (execOrDie) because that the only case the caller doesn't have control over what to do with stdout. In the non-"die" version I'm passing back stdout, and all current callers that care (so not the services test, because we sort | uniq| check some stuff, and only log on failure) seem to go ahead and log it anyway.